### PR TITLE
scripts/build_utils.sh: set hookdir using pbuilderrc

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
being tested at https://shaman.ceph.com/builds/ceph/nautilus-test-3/0410e7c70d2da1bb52b1d26b6092f9eb52503679/

File: scripts/build_utils.sh [hunk 1]
```diff
@@ -490,6 +490,7 @@ env DEBIAN_FRONTEND=noninteractive apt-get update -y -o Acquire::Languages=none
 env DEBIAN_FRONTEND=noninteractive apt-get install -y gnupg
 EOF
     fi
+    chmox +x $hookdir/D04install-updates-repo
```
Comment (djgalloway) [inline scripts/build_utils.sh:493]: chmox

File: scripts/build_utils.sh [hunk 2]
```diff
@@ -616,7 +613,11 @@ setup_pbuilder() {
         # in newer versions. This ticket solves the specific issue in 8.1.1 (which vendors urllib3):
         # https://github.com/shazow/urllib3/issues/567
         echo "USENETWORK=yes" >> ~/.pbuilderrc
+        local hookdir
+        hookdir=$(recreate_hookdir)
```
Comment (djgalloway) [inline scripts/build_utils.sh:617]: should `$()` be double quoted?  genuine question

**Review by djgalloway — CHANGES_REQUESTED** [review]

File: scripts/build_utils.sh (hunk 1 above)
Comment (tchaikov) [reply scripts/build_utils.sh]: @djgalloway fixed and repushed.

File: scripts/build_utils.sh (hunk 2 above)
Comment (tchaikov) [reply scripts/build_utils.sh:617]: no need. but it is advisable to be double quoted when the variable is *referenced* and it should be considered as a single string.

**Comment by dmick** [comment]
1) --hookdir didn't *work*?  2) the reason for doing the updates was internal package inconsistencies, not strictly for ppa, but I'm now thinking that we should not be working from a previously-existing tgz at all, as the hooks are changing it from a "gold" image in ways that we can't control

**Pull request merged**